StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingSaveToLinkVerification/NetworkingSaveToLinkVerificationViewController.swift

```diff
@@ -60,31 +53,38 @@ final class NetworkingSaveToLinkVerificationViewController: UIViewController {
     }
 
     private func showContent(redactedPhoneNumber: String) {
-        let pane = PaneWithHeaderLayoutView(
-            title: STPLocalizedString(
-                "Sign in to Link",
-                "The title of a screen where users are informed that they can sign-in-to Link."
+        let paneLayoutView = PaneLayoutView(
```

Comment:
this seems verify similar to [NetworkingLinkVerificationViewController.swift](https://github.com/stripe/stripe-ios/pull/3180/files#diff-09ac7ebed1da5a704655b3892a8a154ff433c685be6a8d60d8a4c877076cde09), are there anything that could be shared across?

Reply:
yea that is a good point - they were always similar but with V3 they are a bit more similar

there's a chance more can be shared, once the code is finished and I will take another look. Nonetheless, for the most part, the complex logic (OTP logic via reusable `NetworkingOTPView` class) is shared